tests/test_symbolics.py

```diff
@@ -665,7 +667,8 @@ def test_abs_precision(dtype, expected):
     op.apply()
 
     assert expected in str(op)
-    assert np.all(f.data == 1.0)
+    tolerance = 1e-8 if dtype is np.float32 else 1e-16
```

Comment:
Don't need those to be hard coded, you can use `np.finfo(dtype).eps` 

```
In [4]: np.finfo(np.float32).eps
Out[4]: np.float32(1.1920929e-07)
```

Reply:
You can also define those eps at the top for the whole file it's constants (or even in conftest)